theme-utils.mjs

```diff
@@ -1534,12 +1759,16 @@ async function validateThemes( themes, { format, color, tableWidth } ) {
 		}
 
 		const styleCss = await fs.promises.readFile( styleCssPath, 'utf-8' );
-		const themeRequires = getThemeMetadata( styleCss, 'Requires at least', true );
+		const themeRequires = getThemeMetadata(
+			styleCss,
+			'Requires at least',
+			true
+		);
 		const wpVersion = themeRequires
 			? `${ themeRequires }.0`.split( '.', 2 ).join( '.' )
 			: undefined;
-		const hasThemeJsonSupport = wpVersion && semver.valid( `${ wpVersion }.0` ) && semver.gte( `${ wpVersion }.0`, '5.9.0' )
-		const hasThemeJson = fs.existsSync( themeJsonPath );
```

Comment:
I think this has broken the theme validation

Reply:
See this PR https://github.com/Automattic/themes/pull/8116

Reply:
Thanks for catching @MaggieCabrera — it seems there were some changes we thought were formatting, but broke some of the theme validation functions. Fixing in #8174.